### PR TITLE
Completed express-intro Exercise - Eduardo Gallegos

### PR DESCRIPTION
**Pull request by Gallegos22**
Screenshot 2024-04-04 at 12 11 48 PM

### CODE REVIEW

File: express-intro/server.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
```
Comment (BrettAlbright) [inline express-intro/server.ts:9]: Hey Eduardo, make sure to put this use middle ware above the listen method call.

listen should be at the bottom of the server file.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

File: express-intro/server.ts (hunk 1 above)
Comment (Gallegos22) [reply express-intro/server.ts:9]: i just changed it, can you check and let me know if that is better  please and thank you

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**